pkg/metricsservice/utils/tls.go

```diff
@@ -50,8 +53,12 @@ func LoadGrpcTLSCredentials(certDir string, server bool) (credentials.TransportC
 	}
 
 	// Create the credentials and return it
+	minTlsVersion, err := kedatls.GetMinGrpcTlsVersion()
+	if err != nil {
+		ctrl.Log.WithName("grpc_tls_setup").Info(err.Error())
```

Comment:
we should imho return err here

Reply:
I followed the previous behavior that didn't fail the process for invalid value, but selected the default value + returned an error. I can change the function signature to not return an error and perform the logging in the function. wdyt?

Reply:
hm, I am not sure if we shouldn't just fail here. I mean, the grpc connection wouldn't be functional -> KEDA wouldn't be fully functional either.

WDYT @kedacore/keda-maintainers

Reply:
Why not functional? The error only means that it fell back to default value because the configuration value was wrong.

Reply:
Ok. I'd make it fail here but I'll keep the same behavior for the HTTP client

Reply:
this is still pending, correct?